app/src/main/java/com/duckduckgo/app/browser/newtab/NewTabLegacyPageViewModel.kt

```diff
@@ -133,7 +137,11 @@ class NewTabLegacyPageViewModel @Inject constructor(
     // We only want to show New Tab when the Home CTAs from Onboarding has finished
     // https://app.asana.com/0/1157893581871903/1207769731595075/f
     private fun isHomeOnboardingComplete(): Boolean {
-        return dismissedCtaDao.exists(CtaId.DAX_END)
+        val noBrowserCtaExperiment = extendedOnboardingFeatureToggles.noBrowserCtas().isEnabled()
+        return dismissedCtaDao.exists(CtaId.DAX_END) ||
```

Comment:
probably you don't need `dismissedCtaDao.exists(CtaId.DAX_END)` anymore since you added `dismissedCtaDao.exists(CtaId.ADD_WIDGET)`?

Reply:
It's not needed, in my opinion. I didn't remove it as I thought there is some scenario I missed.